hive_setup.sh

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env bash
+set -xe
+
+source logging.sh
+source common.sh
+source utils.sh
+source rhcos.sh
+source ocp_install_env.sh
+
+function generate_hive_manifest() {
```

Comment:
@dgoodwin it would be nice to have your input on whether the data produced by this function looks right.